### PR TITLE
Accounting culture

### PR DESCRIPTION
**Pull request by jazd**
Thank you @mbenninghoven for the translations!

Accounting report results in multiple languages.
New French and Spanish Accounting words in Static Word and Sentence entries.

Change all VIEWs to use I8NWord and I8NSentence VIEWs. Remove ClientCulture from them.

I8NWord and I8NSentence VIEWs are the only ones using ClientCulture() now.

Make Word length longer to accommodate translations of English words.

Test in PostgreSQL only for now:
```SQL
-- Set injected culture for the current session
SELECT ClientCulture() FROM DUAL;
 clientculture 
---------------
          1033
-- Default culture is en-US
SELECT account,name,credit FROM Accounts ORDER BY account;
 account |        name         | credit 
---------+---------------------+--------
       1 | Asset               | f
       2 | Liability           | t
       3 | Income              | t
       4 | Expenses            | f
       5 | Equity              | t
       6 | Equipment           | f
     100 | Cash                | f
     101 | Rent                | f
     102 | Sales               | t
     103 | Equipment           | f
     104 | Loan                | t
     105 | Salary              | f
     106 | Supply              | f
     107 | Petty Cash          | f
     108 | Receivable          | f
     200 | Commissions Payable | t

DELETE FROM inject_culture; INSERT INTO inject_culture (SELECT code FROM culture WHERE name = 'es-MX');
SELECT account,name,credit FROM Accounts ORDER BY account;
 account |            name            | credit 
---------+----------------------------+--------
       1 | la posesión capital        | f
       2 | la obligación              | t
       3 | Ingreso                    | t
       4 | las expensas               | f
       5 | Capital                    | t
       6 | Equipo                     | f
     100 | Dinero en efectivo         | f
     101 | Alquiler                   | f
     102 | Ventas                     | t
     103 | Equipo                     | f
     104 | el empréstito              | t
     105 | Salario                    | f
     106 | Suministro                 | f
     107 | Dinero para gastos menores | f
     108 | Cuenta por cobrar          | f
     200 | Comisiones a pagar         | t

DELETE FROM inject_culture; INSERT INTO inject_culture (SELECT code FROM culture WHERE name = 'fr-FR');
SELECT account,name,credit FROM Accounts ORDER BY account;
 account |        name         | credit 
---------+---------------------+--------
       1 | le capital          | f
       2 | les dettes          | t
       3 | Revenu              | t
       4 | Dépenses            | f
       5 | les fonds           | t
       6 | Équipement          | f
     100 | encaisser           | f
     101 | Louer               | f
     102 | Ventes              | t
     103 | Équipement          | f
     104 | la dette            | t
     105 | le salaire          | f
     106 | La fourniture       | f
     107 | Petite caisse       | f
     108 | Recevable           | f
     200 | Commissions à payer | t
```

### CODE REVIEW

**Review by JacobPenland — APPROVED** [review]
Looks good!

File: Static/1_Sentence.sql [hunk 1]
```diff
@@ -83,7 +83,7 @@ INSERT INTO Sentence (id,culture,value,length) VALUES (92,1033,'Retained Earning
 INSERT INTO Sentence (id,culture,value,length) VALUES (92,1036,'Des bénéfices non répartis',26);
 INSERT INTO Sentence (id,culture,value,length) VALUES (92,2058,'Ganancias retenidas',19);
 INSERT INTO Sentence (id,culture,value,length) VALUES (93,1033,'Discounts',9);
-INSERT INTO Sentence (id,culture,value,length) VALUES (93,1036,'l' escompte',11);
+INSERT INTO Sentence (id,culture,value,length) VALUES (93,1036,'l'' escompte',11);
```
Comment (mbenninghoven) [inline Static/1_Sentence.sql:86]: Is there a reason that you added an extra ' here?

**Review by mbenninghoven — APPROVED** [review]

File: Static/1_Sentence.sql (hunk 1 above)
Comment (jazd) [reply Static/1_Sentence.sql:86]: At least for PostgreSQL, a single quote is escaped in a string by two in a row.  Curious if another character, other than a single quote is valid?

Comment (JacobPenland) [reply Static/1_Sentence.sql:86]: @jazd I believe with postgres, the two single quotes is the correct way to escape in a string. Double quotes are a delimiter identifier.

**Pull request merged**